pkg/neotest/compile.go

```diff
@@ -48,9 +49,10 @@ func CompileSource(t testing.TB, sender util.Uint160, src io.Reader, opts *compi
 	}
 }
 
-// CompileFile compiles a contract from the file and returns its NEF, manifest and hash.
+// CompileFile compiles a contract using contracts caches from the file and returns its NEF, manifest and hash.
```

Comment:
Just move it to a separate sentence. Otherwise it's "using contracts caches from the file".